config/jobs/kubernetes/kops/kops-presubmits.yaml

```diff
@@ -481,9 +481,7 @@ presubmits:
     cluster: k8s-infra-prow-build
     branches:
     - master
-    always_run: false
-    optional: true
-    skip_report: false
+    always_run: true
```

Comment:
Wait, you meat to always run this job? Not sure that's a good idea. Looks more like the role of a periodic test.
We cannot test current kOps changes in the PR with latest k8s build. This will always use the latest kOps marker.

Reply:
Let's revert this part and merge the rest for now.

Reply:
I can change it to run if test/e2e/* is changed

Reply:
perfect